### ISSUE TITLE
Chevrons do not display (chapter 4 of the book) due to incorrect CSS

### ISSUE THREAD

**Issue by wimpog** [senchalabs/jQTouch#43]
In Chapter 4 of "Building iPhone Apps" book on page 53: "Including the arrow class to the li is optional; doing so will add a chevron to the right side of the item in the list." 
I included it, but it does not display.

Here is the CSS for chevrons on li tags (from theme.css:
ul li.arrow {
    background-image: url(img/chevron.png), -webkit-gradient(linear, 0% 0%, 0% 100%, from(#4c4d4e), to(#404142));
    background-position: right center;
    background-repeat: no-repeat;
}

and this CSS overwrites it (has higher preference):
ul.edgetoedge li {
    background: -webkit-gradient(linear, 0% 0%, 0% 100%, from(#1e1f21), to(#272729));
    border-bottom: 2px solid #000;
    border-top: 1px solid #4a4b4d;
    font-size: 20px;
    margin-bottom: -1px;
}

Basically, the latter CSS has higher precedence, and simply sets the background colour.
I also tried opening source files for this book and this chapter, and got the same behaviour. 
My question is, how did you get it to display on screenshots in the book?

Safari 5 (including mobile)
jqtouch-1.0-beta-2-r109
theme: jqt
jquery 1.3.2 (1.4.2)
